manubot/pandoc/util.py

```diff
@@ -59,3 +53,24 @@ def get_pandoc_version() -> Tuple[int, ...]:
         # https://twitter.com/dhimmel/status/1327082301994496000
         raise ImportError("missing pandoc command on system.")
     return pandoc_info["pandoc version"]
+
+def update_command_info(command: str, command_info_dict: dict) -> None:
```

Comment:
I have a slight preference for:

1. naming this function `get_command_info`
2. not taking `command_info_dict` and returning a new dict